skatepark/sass/blocks/_cover.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-cover {
+	padding: clamp(2em, 15%, 8em);
+}
```

Comment:
I think we can do this on theme.json with something like 

```
"core/cover": {
  "spacing": {
	  "padding": {
		  "left": "clamp(2em, 15%, 8em)"
...
	  }
  }
},
```

Reply:
Oh cool, thanks! Will give this a try.